onnxscript/function_libs/torch_aten/ops/core.py

```diff
@@ -19,26 +15,27 @@
 
 from onnxscript import BOOL, INT64
 from onnxscript.function_libs.torch_aten.registration import torch_op
+from onnxscript.function_libs.torch_aten.typing import TFloat, TReal, TTensor
 from onnxscript.onnx_opset import opset18 as op
 from onnxscript.onnx_types import TensorType
 
 
 @torch_op("aten::abs")
-def aten_abs(self):
+def aten_abs(self: TReal) -> TReal:
     # abs(Tensor self) -> Tensor
 
     return op.Abs(self)
 
 
 @torch_op("aten::acos")
-def aten_acos(self):
+def aten_acos(self: TFloat) -> TFloat:
```

Comment:
In current exporter implementation, actually even the input is tensor(bool) it still can work.
If we limit the input to TFloat like this, will it break export?

Reply:
The function does not take bools because onnx Acos does not. The current plan is to use the generated symbolic functions to handle dtype conversion.